sonar-slang-plugin/src/test/java/com/sonarsource/slang/kotlin/KotlinConverterTest.java

```diff
@@ -123,6 +123,11 @@ public void isExpressions() {
     assertTree(kotlinStatement("a is b")).isNotEquivalentTo(kotlinStatement("a is c"));
   }
 
+  @Test
+  public void testEq() {
```

Comment:
Could you please remove this test?

Reply:
Removed this unrelated test in dd8246fe7513ff2430d4fcee43f41aeeb46a82cb